server/repository/src/migrations/v1_05_00/sensor.rs

```diff
@@ -35,7 +35,7 @@ pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
 
             CREATE TABLE temperature_breach (
```

Comment:
Now that I think about it - why are the temperature tables in the `sensor` file?
Would make more sense in `cold_chain.rs`

Reply:
[39fc8aa](https://github.com/openmsupply/open-msupply/pull/2413/commits/39fc8aa473f2b97681c7b13ab05030f7c148db8b)